src/stories/components/JsPsych/JsPsych.js

```diff
@@ -0,0 +1,240 @@
+
+    return () => {
+      document.body.removeChild(jsPsychLibrary);
+      document.body.removeChild(jsPsychStyleSheet);
+      document.body.removeChild(htmlKeyboardResponsePlugin);
+      document.body.removeChild(imageKeyboardResponsePlugin);
+    };
+  }, []);
+
+  useEffect(() => {
+    if (stimuliUrls.length > 0) {
+      setReady(true);
+    }
+  }, [stimuliUrls]);
+
+  // Error handler that prompts the participant to re-click experiment.
+  const errorLoadingJsPsych = (errorCode) => {
+    window.alert('Something went wrong. Please click on experiment again.' +
+      ' Error code: ' + errorCode);
+  };
+
+  // Have to wait for the plugin scripts to be loaded asynchronously before
+  // running the experiment; otherwise it crashes.
+  setTimeout(() => {
+    // Implementation of previous team timeline & trial logic -------------------------- START
+    /* create timeline */
+    const timeline = [];
+
+    /* number of trials */
+    // TODO: Adjust NUMBER_OF_TRIALS back to 199
+    // NOTE: Adjust line below to shorten the number of trials. 199 will go through all 200 iterations.
+    const NUMBER_OF_TRIALS = 5;
+
+    /* define instructions */
+    const instructions = {
+      type: 'html-keyboard-response',
+      stimulus: function() {
+          return "<h1>Instruction</h1>" +
+            "<p>In this experiment, two images will be shown " +
+            "on the screen. Choose the image that looks more like you. </p>" +
+            "<p>Press the letter <strong>E</strong> on the keyboard to select the image on the left.</p>" +
+            "<p>Press the letter <strong>I</strong> on the keyboard to select the image on the right.</p> " +
+            "<p></p>" +
+            "<div style='width: 900px; margin: auto;'>" +
+            // TODO: Figure out how to properly render example images. Both implementations below do not render local images.
+            //        We are able to render via HTTPs URLs but having issues with loading images locally.
+            "<div style='float: left;'><img width='300' src={require('../../../images/example1.png')} alt='Error loading example 1'></img>" +
+            "<p class='small'><strong>Press the E key</strong></p></div>" +
+            "<div class='float: right;'><img width='300' src={ example2 } alt='Error loading example 2'></img>" +
+            "<p class='small'><strong>Press the I key</strong></p></div>" +
+            "</div>" +
+            "<p></p>" +
+            "<p><strong>Press any key to begin.</strong></p>";
+        }
+    };
+    timeline.push(instructions);
+
+    /* generate trials with number of trials */
+    function generateTrials(numberOfTrial) {
+      const trials = [];
+      for (let i = 0; i <= numberOfTrial; i++) {
+        const invFilePath = stimuliUrls[i];
+        const oriFilePath = stimuliUrls[i+1];
+        // TODO: Figure out how to render two images at the same time.
```

Comment:
As mentioned in PR, there is a behavior that is present quite consistently, where invFilePath image loads before oriFilePath image. I believe this is occurring because the page is being read top down, meaning invFilePath has more time to render / is able to render first. Any tips / advice on how to load the images synchronously?

Reply:
(this is also the same behavior from previous teams implementation when they ran experiments)

Reply:
Gotcha. I'm sure we could find a way to make them show up at the same time for an ideal user experience. But for now I think we can proceed as it is.

Reply:
Carlo actually has found a solution to this. Just waiting for this PR to be approved & changes for this will be coming. He added 'jspsych-preload` plugin which helped load the images so both images pop up at the same time!